### ISSUE TITLE
Tesseract cannot find the tessdata file for the second language

### ISSUE THREAD

**Issue by oleg-st** [tesseract-ocr/tesseract#3635]
### Environment

* **Tesseract Version**: 5.0.0-rc1
* **Commit Number**: 8b6390846e726ecb5678d81490e4ec7df35913d5
* **Platform**: Linux 3.10.0-1127.19.1.el7.x86_64

### Current Behavior:

Tesseract cannot find the tessdata file for the second language.

```
/opt/tesseract5/bin/tesseract /tmp/2021-11-10_12-50-02.png /tmp/1 --oem 1 -l 'rus+eng' 
Error opening data file eng.traineddata
Please make sure the TESSDATA_PREFIX environment variable is set to your "tessdata" directory.
Failed loading language 'eng'
Estimating resolution as 145
```

### Expected Behavior:
Tesseract looks for data for the second language in the same directory as for the first.

### Suggested Fix:

It looks like it was broken in this commit d6de055acf6545e316e5326d2e363c23f4763c7c

**Comment by stweil** [comment]
Thank for reporting this regression. This is a regression which must be fixed for 5.0.0. I'll have a look.

**Comment by stweil** [comment]
This issue is fixed in commit 9091055783ac8f0315015c9660d9e625540e0d5e. Please confirm the fix and close this issue.

**Comment by oleg-st** [comment]
It works now. Thank you.
